Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:latest
```

Comment:
Let's not use ubuntu as base. It's huge and way bigger than what we need. I think we should start using the official python base images, especially one of the python:alpine versions.

Reply:
Done